oauth2_provider/views/base.py

```diff
@@ -136,6 +136,18 @@ def form_valid(self, form):
         log.debug("Success url for the request: {0}".format(self.success_url))
         return self.redirect(self.success_url, application)
 
+    def get_active_access_token(self, application, user, scopes):
+        tokens = get_access_token_model().objects.filter(
+            user=user,
+            application=application,
+            expires__gt=timezone.now()
+        ).all()
+
```

Comment:
.all() is redundant here,since you're changing this,can you drop it?

Reply:
done